packages/safepal-wallet/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@near-wallet-selector/safepal-wallet",
+  "version": "9.5.1",
+  "description": "Non-custodial wallet suite trusted by 25M+ users.",
+  "keywords": [
+    "near",
+    "blockchain",
+    "wallets",
+    "dapps",
+    "near-protocol",
+    "near-blockchain",
+    "wallet selector",
+    "injected wallet",
+    "safepal wallet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/near/wallet-selector.git"
+  },
+  "bugs": {
+    "url": "https://github.com/near/wallet-selector/issues"
+  },
+  "homepage": "https://github.com/near/wallet-selector/tree/main/packages/safepal-wallet",
+  "dependencies": {
+    "@near-wallet-selector/core": "workspace:*",
+    "is-mobile": "4.0.0",
+    "near-api-js": "5.0.0"
+  },
```

Comment:
can we use the individual @near-js library here instead near-api-js? we recently migrated the v5 near-api-js to individual @near-js package